feedback.md

```diff
@@ -0,0 +1,25 @@
+# info343-final-project
+
+Our team is proposing the creation of a React-based trivia game web app. We will use one of the available trivia question APIs to create a web-based game with the following functionality:
+
+- User accounts, managed by Google Firebase
+- Sign-in/Sign-out
+- API manipulation, to pull down questions, using the Open Trivia DB (opentdb.com)
+- A timer (10 seconds to answer each question, for example)
+- Leaderboard rankings
```

Comment:
will this include longitudinal rankings? e.g. rankings over time?

Reply:
Our original thought was to provide a strict reckoning of each day's Trivia Challenge that would wipe every day.